Console/UI/Pages/Options.xaml.cs

```diff
@@ -72,19 +59,9 @@ private void btnResetDefault_Click(object sender, RoutedEventArgs e)
             Settings.Default.EnableVerboseLogging = false;
         }
 
-        private void txtUserLocalConfigurationPath_MouseDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
-        {
-            ProcessHelper.StartShellExecutable("explorer.exe", UserLocalConfigurationPath, true);
-        }
-
         private void txtUserConfigurationPath_MouseDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
         {
-            ProcessHelper.StartShellExecutable("explorer.exe", UserConfigurationPath, true);
-        }
-
-        private void txtSharedConfigurationPath_MouseDown(object sender, System.Windows.Input.MouseButtonEventArgs e)
-        {
-            ProcessHelper.StartShellExecutable("explorer.exe", SharedConfigurationPath, true);
+            ProcessHelper.StartShellExecutable("explorer.exe", $"/select \"{Settings.Default.ConfigurationPath}\"", true);

```

Comment:
The "/select" option doesn't open the file:
`ProcessHelper.StartShellExecutable("explorer.exe", $"/select \"{Settings.Default.ConfigurationPath}\"", true);`

This way it works correctly (as before):
ProcessHelper.StartShellExecutable("explorer.exe", $"\"{Settings.Default.ConfigurationPath}\"", true);

Reply:
I pushed this small fix

Reply:
I didn't want the file to directly open, but to be located (hence the select, which you are write, isn't working - I don't know why as it's pretty standard).
Directly opening the file doesn't seem OK for me as I usually use multiple editors depending on my needs. Opening with the default might then be blocking.